woocommerce/woocommerce-ios
# [Card Payments] Add additional configuration keys to Info.plist for Debug builds

Closes #3593 

Please see [this comment](https://github.com/woocommerce/woocommerce-ios/pull/3656#issuecomment-778434134) for context.

We need to provide several keys in the Info.plist file in order for the Stripe Terminal SDK to work. For more info, please see: https://stripe.com/docs/terminal/sdk/ios#configure

This PR adds a new build phase that would merge the contents of an Info.plist file with entries specific for a given build configuration into the final Info.plist

## Changes
* Add AdditionalKeys-Debug-Info.plist with keys specific for the integration with the Stripe Terminal SDK
* Add a Run Script Build Phase that will merge these additional keys with the content of Info.plist into a new file (that is added to .gitignore)
* Change the base build configuration to point to the Info.plist file that contains the merged data

## How to test
* Checkout the branch, build and run in debug mode. Check the there is no local diff
* See this comment for detailed testing instructions: https://github.com/woocommerce/woocommerce-ios/pull/3656#issuecomment-778485222

Update release notes:
